code-scanning/endorlabs.yml

```diff
@@ -0,0 +1,46 @@
+name: Endor Labs Scan
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+  schedule:
+    - cron: $cron-weekly
+jobs:
+  scan:
+    permissions:
+      security-events: write # Used to upload sarif artifact to GitHub
+      contents: read # Used to checkout a private repository but actions/checkout. 
+      actions: read # Required for private repositories to upload sarif files. GitHub Advanced Security licenses are required.
+      id-token: write # Used for keyless authentication to Endor Labs
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    #### Package Build Instructions
+    ### Use this section to define the build steps used by your software package. 
+    ### Endor Labs does this for you when possible. 
+    # - name: Setup Java
+    #   uses: actions/setup-java@v3
```

Comment:
Should this be commented out?

Reply:
@jtamsut - Yes, this is intended to demonstrate the setup of tools and optional build steps for a software package, that may vary.